main.py

```diff
@@ -1,11 +1,43 @@
 import init_django_orm  # noqa: F401
 
+import json
+from django.utils.timezone import now
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race_data = player_info["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data["description"]}
+        )
+
+        for skill_data in race_data["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"], "race": race}
+            )
 
+        guild_data = player_info["guild"]
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data["description"]}
+            )
 
-if __name__ == "__main__":
-    main()
+        Player.objects.get_or_create(
+            nickname=player_name,
+            defaults={
+                "email": player_info["email"],
+                "bio": player_info["bio"],
+                "race": race,
+                "guild": guild,
+                "created_at": now(),
```

Comment:
The `created_at` field should be handled automatically by Django. You should use `auto_now_add=True` in your model definition instead of manually setting the field here. This way, Django will automatically set `created_at` to the current date when an object is first created. (Checklist point 4.2)